mava/utils/config.py

```diff
@@ -18,27 +18,59 @@
 from omegaconf import DictConfig
 
 
+def check_sebulba_config(config: DictConfig) -> None:
+    """Checks that the given config does not have conflicting values."""
+    assert (
+        config.system.num_updates > config.arch.num_evaluation
+    ), "Number of updates per evaluation must be less than total number of updates."
+    config.system.num_updates_per_eval = config.system.num_updates // config.arch.num_evaluation
+
+    assert config.arch.num_envs % len(config.arch.learner_device_ids) == 0, (
+        "Number of environments must be divisible by the number of learner."
+        + "The output of each actor is equally split across the learners."
+    )
+
+    num_eval_samples = (
+        int(config.arch.num_envs / len(config.arch.learner_device_ids))
+        * config.system.rollout_length
+    )
+    assert num_eval_samples % config.system.num_minibatches == 0, (
+        f"Number of training samples per evaluator ({num_eval_samples})"
+        + f"must be divisible by num_minibatches ({config.system.num_minibatches})."
+    )
+
+
 def check_total_timesteps(config: DictConfig) -> DictConfig:
     """Check if total_timesteps is set, if not, set it based on the other parameters"""
-    n_devices = len(jax.devices())
+
+    if config.arch.architecture_name == "anakin":
+        n_devices = len(jax.devices())
+        update_batch_size = config.system.update_batch_size
+        n_accumulate = 1  # We dont accumulate envs in anakin
+    else:
+        n_devices = 1  # We only use a single device's output when updating.
+        update_batch_size = 1
+        n_accumulate = config.arch.n_learner_accumulate
```

Comment:
I believe the current setup is clearer because the learner thread uses the output from a single actor. The `num_env `parameter kind of represents the number of environments that are processed in each update.

Using a single device ties in with this approach: as each update step relies on the output from a single actor, which is then distributed across the learner devices. Adding more devices helps to speed up trajectory collection and parameter updating since the workload is shared, but it does not increase the number of timesteps.